src/h3lib/include/coordijk.h

```diff
@@ -108,7 +108,7 @@ Direction _rotate60ccw(Direction digit);
 Direction _rotate60cw(Direction digit);
 int ijkDistance(const CoordIJK *a, const CoordIJK *b);
 void ijkToIj(const CoordIJK *ijk, CoordIJ *ij);
-void ijToIjk(const CoordIJ *ij, CoordIJK *ijk);
+H3Error ijToIjk(const CoordIJ *ij, CoordIJK *ijk);
```

Comment:
This makes me suspicious that we may need something similar for `ijkToIj` or the cube functions as well, might be worth adding tests.

Reply:
I am not sure if ijkToIj can be overflowed assuming that the input coordinates are all non-negative.